Lib/concurrent/futures/_base.py

```diff
@@ -544,18 +545,40 @@ def map(self, fn, *iterables, timeout=None, chunksize=1):
         """
```

Comment:
Can you double check what behavior you want for this in the case of `ProcessPoolExecutor`? For `prefetch = 10`, I think you'll be prefetching 10 chunks at the moment, which is a lot more than you probably meant.

Reply:
As noted above, I really do want to prefetch 10 chunks; the `prefetch` is about how many "things to occupy a single worker as an atomic unit" (which the `chunksize` argument describes as a "task") to have outstanding above and beyond the number of tasks we expect to actually be running at any given time. Increasing the `chunksize` should not increase the risk of a worker needlessly falling idle due to lack of outstanding tasks.